ci-operator/step-registry/openshift/e2e/libvirt/setup/openshift-e2e-libvirt-setup-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -euo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+INSTANCE_PREFIX="${NAMESPACE}"-"${JOB_NAME_HASH}"
+
+mkdir -p "${HOME}"/.ssh
+mock-nss.sh
+
+# gcloud compute will use this key rather than create a new one
+cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
+chmod 0600 "${HOME}"/.ssh/google_compute_engine
+cp "${CLUSTER_PROFILE_DIR}"/ssh-publickey "${HOME}"/.ssh/google_compute_engine.pub
+
+# Copy pull secret to user home
+cp "${CLUSTER_PROFILE_DIR}"/pull-secret "${HOME}"/pull-secret
+
+gcloud auth activate-service-account --quiet --key-file "${CLUSTER_PROFILE_DIR}"/gce.json
+gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
+gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
+
+
+LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
+  --quiet \
+  --project "${GOOGLE_PROJECT_ID}" \
+  --zone "${GOOGLE_COMPUTE_ZONE}" \
+  --recurse /bin/openshift-install packer@"${INSTANCE_PREFIX}":/home/packer/openshift-install
```

Comment:
When [this](https://github.com/ironcladlou/openshift4-libvirt-gcp/pull/33) merges and images are updated, this can go away #L25-29

Reply:
nm, all set